spread_tests/suites/termus/task_boot_firmware/task.yaml

```diff
@@ -0,0 +1,13 @@
+
+summary: Termus boot-firmware command
+kill-timeout: 2m
+
+execute: |
+    ${SCRIPTS_DIR}/start_termus.sh
+    ${SCRIPTS_DIR}/pebble_wrapper.sh stop dummy
+    ${SCRIPTS_DIR}/pebble_wrapper.sh services | grep -E -e "dummy\s+enabled\s+inactive"
+    ${SCRIPTS_DIR}/pebble_wrapper.sh changes | grep -e "Stop service \"dummy\""
```

Comment:
Instead of running the client from inside the container, I think we should have a ```kernos-client``` script that runs it locally in the VM instead. It will require access to a firmware payload for ```firmware-install```, and in general I think the test pattern to match the host pc run the client , device (container) runs the daemon is a good start.